src/classes/Table/types.ts

```diff
@@ -63,7 +64,7 @@ export type $QueryOptions<
 > = $ReadOptions<Execute, Parse> &
   TableReadOptions & {
     reverse: boolean
-    select: DocumentClient.Select
+    select: Select | `${Select}` | Lowercase<Select>
```

Comment:
why don't we keep it with the `Select` only?

Reply:
same as above